tests/Integration/Behaviour/Features/Context/Domain/ModuleFeatureContext.php

```diff
@@ -62,7 +64,7 @@ public function assertModuleInfos(string $technicalName, TableNode $tableNode):
             if (isset($data['installed'])) {
                 Assert::assertEquals(PrimitiveUtils::castStringBooleanIntoBoolean($data['installed']), $moduleInfos->isInstalled());
             }
-        } catch (ModuleNotFoundException $e) {
+        } catch (ModuleNotFoundException | ModuleNotInstalledException $e) {
```

Comment:
As I commented above, fetching the data of an installed (BUT present) module is a valid use case